pkg/clicmd/init.go

```diff
@@ -174,22 +178,22 @@ func newInitCmd(fs afero.Fs, wd string) *cobra.Command {
 	return initCmd
 }
 
-func genKsRoot(appName, ksDir, wd string) (string, error) {
-	if ksDir == "" {
-		return "", errors.New("invalid working directory")
+func genKsRoot(appName, ksExecDir, newAppDir string) (string, error) {
```

Comment:
A comment with examples of the different ways of calling would be helpful to me.

Reply:
i will give an example or two, but i think the tests do a good job of showing all the cases?